kafkaesk/metrics.py

```diff
@@ -0,0 +1,36 @@
+import prometheus_client as client
+
+NOERROR = "none"
+
+PUBLISHED_MESSAGES = client.Counter(
+    "published_messages", "Number of published messages", ["stream_id", "partition", "error"]
+)
+
+
+CONSUMED_MESSAGES = client.Counter(
+    "consumed_messages",
```

Comment:
I would have prefixed these metrics with the `kafkaesk` suffix for avoiding any collision with other metric names used by the users.

Reply:
We can update!